coldfront/plugins/qumulo/tests/test_tasks.py

```diff
@@ -245,3 +260,284 @@ def test_conditionally_update_storage_allocation_statuses_checks_all_pending_all
             self.assertEqual(
                 conditionally_update_storage_allocation_status_mock.call_count, 2
             )
+
+
+class TestIngestQuotasWithDailyUsages(TestCase):
+    def setUp(self) -> None:
+        self.original_storage_path = os.environ.get("STORAGE2_PATH")
+        self.STORAGE2_PATH = "/storage2/fs1"
+        os.environ["STORAGE2_PATH"] = self.STORAGE2_PATH
+
+        self.client = Client()
+        build_data = build_models()
+
+        self.project = build_data["project"]
+        self.user = build_data["user"]
+
+        self.quotas = self.__get_qumulo_quota_data(
+            get_mock_quota_data(self.STORAGE2_PATH)
+        )
+
+        self.status_active = AllocationStatusChoiceFactory(name="Active")
+        self.status_ready_for_deletion = AllocationStatusChoiceFactory(
+            name="Ready for Deletion"
+        )
+
+        for index, (path, value) in enumerate(
+            get_mock_quota_data(self.STORAGE2_PATH).items()
+        ):
+            if "exclude" in path:
+                continue
+
+            form_data = {
+                "storage_filesystem_path": path.rstrip("/"),
+                "storage_export_path": path.rstrip("/"),
+                "storage_name": f"for_tester_{index}",
+                "storage_quota": value["limit"],
+                "protocols": ["nfs"],
+                "rw_users": [f"user_{index}_rw"],
+                "ro_users": [f"user_{index}_ro"],
+                "storage_ticket": f"ITSD-{index}",
+                "cost_center": "Uncle Pennybags",
+                "department_number": "Time Travel Services",
+                "service_rate": "general",
+            }
+
+            # create_allocation(project=self.project, user=self.user, form_data=form_data)
+            allocation = create_allocation(
+                project=self.project, user=self.user, form_data=form_data
+            )
+            allocation.status = self.status_active
+            allocation.save()
+
+        self.storage_filesystem_path_attribute_type = (
+            AllocationAttributeType.objects.get(name="storage_filesystem_path")
+        )
+        self.storage_quota_attribute_type = AllocationAttributeType.objects.get(
+            name="storage_quota"
+        )
+
+        return super().setUp()
+
+    def tearDown(self) -> None:
+        os.environ["STORAGE2_PATH"] = self.original_storage_path
```

Comment:
@harterjwustl What are the reasons for not using mock?

Reply:
I believe (and my testing shows) this only mocks it locally.  Since the mock data is being being build elsewhere, it isn't getting the same data.

Reply:
@harterjwustl I am curious why mocking did not change the data in the test. The test_list_qumulo_quota_usages shows how mocking with an environment variable works. In particular, run the following test.

https://github.com/WashU-IT-RIS/coldfront-wustl-fork/blob/0a984b275d6e661db6be2cd7b8032474f46d8376/coldfront/plugins/qumulo/tests_integration/utils/test_qumulo_api/test_list_qumulo_quota_usages.py#L53

Reply:
I am not totally sure, either, but it seems to have something to do with it being loaded in different files.  I tried your suggestion, but it broke the tests.  some of the data used what I loaded, and some of it used what was in my .env file.

I might be able to do it by making a mock for each file I'm using that uses it, but it seems hard to track and adds complexity.